int16 FAILURE = -1
int16 WAITING_FOR_STATE = 1
int16 GOTO_STANDBY_POSE = 2
int16 GOTO_STANDBY_POSE_DONE = 3

int16 START_JOINT1_JOG = 4
int16 START_JOINT1_JOG_STOP = 5

int16 ARUCO_MARKER_FOUND = 6
int16 ARUCO_MARKER_NOT_FOUND = 7

int16 MOVE_TO_CLOSER_ARUCO_VIEWPOINTS = 8
int16 MOVE_TO_CLOSER_ARUCO_VIEWPOINTS_DONE = 9

int16 AVERAGE_ARUCO_POSE = 10
int16 AVERAGE_ARUCO_POSE_DONE = 11

int16 TRANSFORM_MARKER2ROOT = 12
int16 TRANSFORM_MARKER2ROOT_DONE = 12

int16 HEADON_VIEWPOINTS_PLANNING = 13
int16 HEADON_VIEWPOINTS_PLANNING_SUCCESS = 14
int16 HEADON_VIEWPOINTS_PLANNING_FAIL = 15

int16 HEADON_VIEWPOINTS_MOVING = 16
int16 HEADON_VIEWPOINTS_MOVING_DONE = 17

int16 POINTCLOUD_PAIR_CASE1 = 18
int16 POINTCLOUD_PAIR_CASE2 = 19
int16 POINTCLOUD_PAIR_CASE3 = 20

int16 EXECUTE_ICP = 21
int16 EXECUTE_ICP_DONE = 22

int16 CAL_HEIGHT = 23
int16 CAL_HEIGHT_DONE = 23

int16 ADD_OB_HEIGHT = 24
int16 ADD_OB_HEIGHT_DONE = 24

int16 OB_HEIGHT_EXCEED = 25
int16 OB_HEIGHT_NOT_EXCEED = 26

int16 CASE1_START_COPY_CLOUD = 27
int16 CASE1_START_COPY_CLOUD_DONE = 28

int16 CASE3_TERMINATION = 29

int16 POINTCLOUD_PREPROCESSING_DONE = 30

int16 LEVEL2CONTROL_START = 31

int16 EXECUTE_SEA3 = 32
int16 EXECUTE_SEA3_DONE = 33

int16 RATIO_NOT_ENOUGH = 34
int16 RATIO_ENOUGH = 35

int16 ADD_ADJACENNT_HEIGHT_DONE = 36

int16 EXIST_OCCLUDING = 37
int16 NOT_EXIST_OCCLUDING = 38

int16 CIRCULAR_BF_VIEWPOINTS_PLANNING_DONE = 39
int16 CIRCULAR_BF_VIEWPOINTS_MOVING_DONE = 40

int16 RESCAN_SEA = 41

int16 CIRCULAR_BF_VIEWPOINTS_EXISTS = 42
int16 CIRCULAR_BF_VIEWPOINTS_NOT_EXISTS = 43
int16 ADJACENT_HEIGHT_EXCEED = 44
int16 ADJACENT_HEIGHT_NOT_EXCEED = 45
int16 IK_FAILURE = 46

std_msgs/Header header
int16 status
---
int16 error_code